build-wc.js

```diff
@@ -8,12 +8,19 @@ const build = async () => {
     "./dist/video-player-wc/scripts.js",
     "./dist/video-player-wc/vendor.js",
     "./dist/video-player-wc/main.js",
+    "web-component/assets/videojs-markers.js",
```

Comment:
Before adding they should be minified and uglified to reduce final JS output file size

Reply:
Done

Reply:
Wha is the change in the JS size post this change

Reply:
Previous js file size: 2.74 MB
Current js file size: 2.72 MB